api/envoy/api/v2/auth/cert.proto

```diff
@@ -235,12 +235,24 @@ message CommonTlsContext {
   // Configs for fetching TLS certificates via SDS API.
   repeated SdsSecretConfig tls_certificate_sds_secret_configs = 6;
 
+  // Config for validation context via SDS API.
+  message ValidationContextSdsConfig {
+    // An optional list of Subject Alternative Names. If specified, Envoy will ignore
+    // CertificateValidationContext.verify_subject_alt_name received from SDS API,
+    // and will use this list to verify that the Subject Alternative Name of the presented
+    // certificate matches one of the specified values.
+    repeated string override_verify_subject_alt_name = 1;
+
+    // Config for fetching validation context via SDS API.
+    SdsSecretConfig sds_secret_config = 2;
```

Comment:
change this to be the first field as index =1 since it is the most important field

Reply:
Done.